coreutils (5.2.1-2.kon.1) local; urgency=low

  * Local version!
  * src/uname.c (main): Don't recognize -i and -p at all.  (See: #193170)

 -- Kalle Olavi Niemitalo <kon@iki.fi>  Sat, 30 Apr 2005 12:38:13 +0300

--- coreutils_5.2.1-2/debian/patches/21_uname-no-unknowns.diff	2005-04-30 12:21:04.000000000 +0300
+++ coreutils_5.2.1-2.kon.1/debian/patches/21_uname-no-unknowns.diff	2005-04-30 12:24:40.000000000 +0300
@@ -18,6 +19,15 @@
    -o, --operating-system   print the operating system\n\
  "), stdout);
        fputs (HELP_OPTION_DESCRIPTION, stdout);
+@@ -158,7 +154,7 @@
+ 
+   atexit (close_stdout);
+ 
+-  while ((c = getopt_long (argc, argv, "asnrvmpio", long_options, NULL)) != -1)
++  while ((c = getopt_long (argc, argv, "asnrvmo", long_options, NULL)) != -1)
+     {
+       switch (c)
+ 	{
 @@ -189,14 +185,6 @@
  	  toprint |= PRINT_MACHINE;
  	  break;
